### DISCUSSION TITLE
Web Configuration won’t start - keeps timing out.

### DISCUSSION THREAD

**Discussion by Firemanjoe** [hyperion-project/hyperion.ng#1322] [Q&A] [answered]
Hello, 

I wanted to post into the forum; however, for some reason, it is not recognizing my GitHub log in. 

Eitherway, what I am wondering is if there a command that I can put into my pi to have the web configuration enabled? My cat ended up turn off my pi (this is being corrected without kitty harm) and I have not been able to get back into my web configator to make adjustments. I am able to turn it on & off via home kit/homebrige, but for some reason, i am unable to access the site. 

Suggestions? 

E: it appears that it’s limited to my ipad as I was able to log into via my MacBook Pro.

**Comment by Lord-Grey** [comment, marked as answer]
Question was resolved by user.
